RestSharpClient/Model/Response/RestfulBooker/BookingIds.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <auto-generated />
+//
+// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
+//
+//    using RestfulBooker;
+//
+//    var bookings = Bookings.FromJson(jsonString);
+
+namespace RestSharpClient.Model.RestfulBooker
+{
+    using System;
+    using System.Collections.Generic;
+
+    using System.Globalization;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    public partial class BookingIds
```

Comment:
Why are all these classes partial?

Reply:
Was having individual Serialization actions specific to models split as separte partial classes. Later removed them all and kept the serialization actions in common. Failed to remove them later. Removed 'partial' from all not required places. Thanks!